internal/bundle/bundles/python_wheel_task_with_cluster/databricks_template_schema.json

```diff
@@ -0,0 +1,25 @@
+{
+    "properties": {
+        "project_name": {
+            "type": "string",
+            "default": "my_test_code",
+            "description": "Unique name for this project"
+        },
+        "spark_version": {
+            "type": "string",
+            "description": "Spark version used for job cluster"
+        },
+        "node_type_id": {
+            "type": "string",
+            "description": "Node type id for job cluster"
+        },
+        "unique_id": {
+            "type": "string",
+            "description": "Unique ID for job name"
+        },
+        "instance_pool_id": {
```

Comment:
This is not being used in the test, should we use it in the cluster specification to reduce the time this test takes to run?

Reply:
We create an interactive cluster as part of this test, so instance_pool_id needs to be removed